dotplayplay/frontend
# Enhance Slider Component with Auto-Betting and Sound Effects

This pull request introduces several enhancements to the existing slider component, adding auto-betting functionality, dynamic styling, and sound effects. These changes improve the user experience and expand the component's capabilities.

**Changes Proposed:**

1. **Auto-Betting Functionality:**
   - Added a toggleable auto-betting mode that automatically generates random steps at regular intervals.
   - Users can start and stop auto-betting with a button click.
   - Sound effects play on each auto-bet result, enhancing user engagement.
   - The slider control is disabled when the `start auto button` is clicked

2. **Dynamic Styling:**
   - Slider color now changes dynamically based on the user's selected value.
   - The slider's appearance reflects the probability of winning and losing.

3. **Sound Effects:**
   - Added sound effects for slider interaction.
   - A sound plays when the component is mounted for a more immersive experience.

4. **Improved Code Organization:**
   - Reorganized the code for better readability and maintainability.
   - Removed unnecessary code and added comments for clarity.

5. **Default View:**
   - Set the "Manual" view as the default view when the component is initially loaded.

**How to Test:**

1. Run the application with these changes.
2. Verify that the slider color changes dynamically as the user moves the slider.
3. Test the auto-betting functionality by starting and stopping it.
4. Ensure that sound effects play as expected during interactions.
5. Verify that the "Manual" view is displayed by default when the component loads.
